packages/client-core/src/social/services/FriendService.ts

```diff
@@ -30,9 +30,9 @@ const FriendState = defineState({
 
 export const FriendServiceReceptor = (action) => {
   getState(FriendState).batch((s) => {
-    let newValues, selfUser, otherUser, otherUserId
     matches(action)
       .when(FriendAction.loadedFriendsAction.matches, (action) => {
+        let newValues
```

Comment:
lets clean up all of these as well. this can be `s.friends.friends.set(action.friends.data)`

Reply:
Tried to clean up all inner scope variables however the create and patch friend action seems not clear to me what the first author tried to implement.

Reply:
thats alright, this is all broken anyway. just want it cleaned up where it makes sense to